test/src/main/java/org/axonframework/test/aggregate/ResultValidator.java

```diff
@@ -249,6 +256,15 @@
      */
     ResultValidator<T> expectScheduledDeadlineOfType(Instant scheduledTime, Class<?> deadlineType);
 
+    /**
+     * Asserts that a deadline with the given {@code deadlineName} has been scheduled after the given {@code duration}.
```

Comment:
Should be `@code scheduledTime`
Also the wording for it might change a bit since this is the expected time for the deadline and not the duration anymore.

Reply:
Good catch, thanks for that! Will update this line of javadoc accordingly.